fs/src/jsMain/kotlin/externals/kotlin-multiplatform-projects-fs/src/index.module_ncp.kt

```diff
@@ -0,0 +1,36 @@
+@file:Suppress("INTERFACE_WITH_SUPERCLASS", "OVERRIDING_FINAL_MEMBER", "RETURN_TYPE_MISMATCH_ON_OVERRIDE", "CONFLICTING_OVERLOADS")
```

Comment:
For my knowledge, is this file created automatically by some tooling? Like generating the cinterop header file.

Reply:
yeah it was generated with [dukat](https://github.com/Kotlin/dukat), but usually the externals it generates are full of errors and need to be modified, which is what i did here.

if you want to try it out yourself you can add `generateExternals` to an npm dependency in `build.gradle.kts` like so:
```kotlin
implementation(npm("ncp", "2.0.0", generateExternals=true))
```
then run `gradle jsGenerateExternals` and it should put them in the build directory.